packages/components/src/Typeahead/Typeahead.test.js

```diff
@@ -177,6 +198,41 @@ describe('Typeahead', () => {
 			// then
 			expect(onSelect).toBeCalled();
 		});
+
+		it('should render typeahead selected item by id', () => {
+			// given
+			const props = {
+				...initialProps,
+				value: 'le title 1',
+				valueId: 'letitle1copy',
+				items: itemsObjectWithId,
+			};
+
+			// when
+			render(<Typeahead {...props} />);
+
+			// then
+			const titleList = screen.getAllByTitle('le title 1');
+			expect(titleList[0]).not.toHaveClass('theme-selected');
+			expect(titleList[1]).toHaveClass('theme-selected');
+		});
+
+		it('should render typeahead selected item by title', () => {
+			// given
+			const props = {
+				id: 'my-search',
+				value: 'le title 1',
+				items: itemsObjectWithId,
+			};
+
+			// when
+			render(<Typeahead {...props} />);
+
+			// then
+			const titleList = screen.getAllByTitle('le title 1');
+			expect(titleList[1]).toHaveClass('theme-selected');
+			expect(titleList[1]).toHaveClass('theme-selected');
```

Comment:
```suggestion
			expect(titleList[0]).not.toHaveClass('theme-selected');
			expect(titleList[1]).toHaveClass('theme-selected');
```

Reply:
Thanks , didnt notice
Pushed a new commit about this , in fact selection by title should select both items like it's already to case